### PR TITLE
feat: remove usage of user privacy field [WPB-10025]

### PR DESCRIPTION
**Pull request by aweiss-dev**
We need to remove the send anonymous usage data and crash reports checkbox.
We should use the telemetry_sharing attribute in the future for everything related to data sharing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,10 @@
     "@lexical/history": "0.16.1",
     "@lexical/react": "0.16.1",
     "@mediapipe/tasks-vision": "0.10.14",
+    "@wireapp/api-client": "^27.1.0",
```
Comment (V-Gira) [inline package.json]: don't think api-client should show up here, isn't it handled by core?

Comment (aweiss-dev) [reply package.json]: That is true. Gonna remove it

File: src/script/page/MainContent/panels/preferences/accountPreferences/DataUsageSection.tsx [hunk 2]
```diff
@@ -60,31 +59,21 @@ const DataUsageSection: React.FC<DataUsageSectionProps> = ({
...
-      setOptionPrivacy(settings.privacy.improve_wire);
       setOptionTelemetry(settings.privacy.telemetry_sharing);
     };
     amplify.subscribe(WebAppEvents.PROPERTIES.UPDATED, updateProperties);
     return () => amplify.unsubscribe(WebAppEvents.PROPERTIES.UPDATED, updateProperties);
   }, []);
 
+  const showCountly = !isTeam || (isTeam && isCountlyEnabled);
+
+  if (!showCountly && !isActivatedAccount) {
```
Comment (V-Gira) [inline src/script/page/MainContent/panels/preferences/accountPreferences/DataUsageSection.tsx:70]: Looks like it should be an `OR` at first glance, is that not the case?

Comment (aweiss-dev) [reply src/script/page/MainContent/panels/preferences/accountPreferences/DataUsageSection.tsx:70]: No. If both statements are falsy, the component should return null

If one of them is truthy, we want to render the component

**Review by przemvs — APPROVED** [review]

**Pull request merged**